lib/src/Utilities.cc

```diff
@@ -1034,6 +1034,35 @@ char *getHttpFullDate(const trantor::Date &date)
     return lastTimeString;
 }
 
+void dateToCustomFormattedString(const std::string &fmtStr,
+                                 std::string &str,
+                                 const trantor::Date &date)
+{
+    auto nowSecond = date.microSecondsSinceEpoch() / MICRO_SECONDS_PRE_SEC;
+    time_t seconds = static_cast<time_t>(nowSecond);
+    struct tm tm_LValue = date.tmStruct();
+    std::stringstream Out;
+    Out.imbue(std::locale{"en_US"});
+    Out << std::put_time(&tm_LValue, fmtStr.c_str());
+    str = Out.str();
+}
+
+const std::string getHttpFullDateStr(const trantor::Date &date)
```

Comment:
`&` is missed.

Reply:
Oh, sorry, I forgot the string was `static thread_local`. Indeed it should be returning the address type. Will fix now.

Reply:
I've re-run [Build & Test](https://github.com/easymoney322/drogon/actions/runs/12153373925) job.